device/mellanox/x86_64-mlnx_msn2700-r0/platform_reboot

```diff
@@ -31,4 +31,6 @@ if [[ "${EXIT_CODE}" != "${EXIT_SUCCESS}" ]]; then
     fi
 fi
 
-exec /sbin/reboot $@
+echo 1 > /bsp/system/pwr_cycle
```

Comment:
Is it possible to fail? If you stop the BSP service, the file will disappear, and finally reboot will not work.

Reply:
Good point, changed to use the sysfs path itself.